src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1697,7 +1696,18 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
             _terminal->SetCursorOn(true);
             _cursorTimer.value().Start();
         }
-        _rowsToScroll = _settings.RowsToScroll();
+
+        _UpdateSystemParameterSettings();
+    }
+
+    // Method Description
+    // - Updates internal params based on system parameters
+    void TermControl::_UpdateSystemParameterSettings() noexcept
+    {
+        if (!SystemParametersInfoW(SPI_GETWHEELSCROLLLINES, 0, &_rowsToScroll, 0))
+        {
+            _rowsToScroll = 3;

```

Comment:
what the heck, why 3?

Reply:
3 appears to be the default value configured for this feature on Windows

Reply:
I believe SystemParamInfoW will never fail, but it _can_ fail, so why not protecc

Reply:
![excuse-me](https://user-images.githubusercontent.com/18356694/87334971-24bb8980-c505-11ea-9ef9-cf9901cfc8c9.gif)

Reply:
documented, but yeah i know